setup/www/ansible-playbook.yaml

```diff
@@ -78,20 +102,36 @@
       command: "bash -c '{{ update_command }}'"
       tags: setup
 
+    ### configure cron and utility capabilities
+
+    - name: Configure | setup cron for nightly
+      cron: name="promote_nightly" minute="*/5" job="/home/staging/promote/promote_nightly.sh" user="{{ dist_user }}"
+      tags: setup
+
+    - name: Configure | setup cron for download-stats
+      cron: name="update_download_stats" minute="30" hour="1" job="/home/iojs/update-download-stats.sh" user="root"
+      tags: setup
```

Comment:
both of these crontab jobs go into the user specific crontab (i.e. `crontab -e`) and not into `/etc/crontab`. Not sure if that matters.